ui-components/src/helpers/tableSummaries.tsx

```diff
@@ -1,81 +1,135 @@
 import * as React from "react"
 import { t } from "./translator"
-import { UnitSummary } from "@bloom-housing/backend-core/types"
+import { UnitSummary, UnitsSummary } from "@bloom-housing/backend-core/types"
 import { GroupedTableGroup } from "../tables/GroupedTable"
 
-export const unitSummariesTable = (summaries: UnitSummary[]) => {
-  const unitSummaries = summaries.map((unitSummary) => {
-    const minIncome =
-      unitSummary.minIncomeRange.min == unitSummary.minIncomeRange.max ? (
-        <strong>{unitSummary.minIncomeRange.min}</strong>
-      ) : (
-        <>
-          <strong>{unitSummary.minIncomeRange.min}</strong> {t("t.to")}{" "}
-          <strong>{unitSummary.minIncomeRange.max}</strong>
-        </>
-      )
+export const getSummaryRow = (
+  minIncomeRangeMin?: string,
+  minIncomeRangeMax?: string,
+  rentRangeMin?: string,
+  rentRangeMax?: string,
+  rentAsPercentIncomeRangeMin?: string,
+  rentAsPercentIncomeRangeMax?: string,
+  totalAvailable?: number,
+  totalCount?: number,
+  unitTypeName?: string
+) => {
+  let minIncome = <></>
+  if (minIncomeRangeMin == undefined) {
+    //TODO(#345): figure out what to display when there's no data
+    minIncome = <strong>{t("t.call")}</strong>
+  } else if (minIncomeRangeMin == minIncomeRangeMax) {
+    minIncome = (
+      <strong>
+        {minIncomeRangeMin}
+        {t("t.perMonth")}
+      </strong>
+    )
+  } else {
+    minIncome = (
+      <>
+        <strong>{minIncomeRangeMin}</strong> {t("t.to")}{" "}
+        <strong>
+          {minIncomeRangeMax} {t("t.perMonth")}
+        </strong>
+      </>
+    )
+  }
```

Comment:
I think we'll want to be able to support having one of min or max (@anders-schneider lmk if I'm mistaken). Can you update to something like this:
```
if (minIncomeRangeMin == undefined && minIncomeRangeMax == undefined) {
// no data
} else if (minIncomeRangeMin == minIncomeRangeMax || minIncomeRangeMax == undefined) {
// display only min
} else if (minIncomeRangeMin == undefined) {
// display only max
} else {
// format range
}
```
Same idea for `getRent` below.

Reply:
updated both